routing/bicycle_directions.cpp

```diff
@@ -175,9 +180,13 @@ void BicycleDirectionsEngine::Generate(RoadGraphBase const & graph, vector<Junct
     // prevJunction == path[i - 1] and currJunction == path[i].
     if (inFeatureId.IsValid())
       LoadPathGeometry(uniNodeId, {prevJunction, currJunction}, pathSegment);
-    pathSegment.m_trafficSegs = {{inFeatureId.m_index, static_cast<uint16_t>(inSegId),
-                                  inIsForward ? TrafficInfo::RoadSegmentId::kForwardDirection
-                                              : TrafficInfo::RoadSegmentId::kReverseDirection}};
+
+    if (m_numMwmIds)
+    {
+      NumMwmId const numMwmId =
+          m_numMwmIds->GetId(inFeatureId.m_mwmId.GetInfo()->GetLocalFile().GetCountryFile());
```

Comment:
For documentation, please, put an assertion here that inFeatureId.m_mwmId().IsAlive().